validator/node/node.go

```diff
@@ -368,6 +372,8 @@ func (c *ValidatorClient) registerPrometheusService(cliCtx *cli.Context) error {
 }
 
 func (c *ValidatorClient) registerValidatorService(cliCtx *cli.Context) error {
+	jsonValidator := validator.New()
```

Comment:
validator.New() sounds like a big conflict with the concept of an eth2 validator. Maybe we should alias the package to jsonValidation, so you can do jsonValidator := jsonValidation.New()

Reply:
You can probably move this to right above where it is used, above line 413